v3/src/components/app.tsx

```diff
@@ -100,6 +100,19 @@ export const App = observer(function App() {
           addDefaultComponents()
         }
       }
+
+      const { di } = urlParams
+      if (typeof di === "string") {
+        // setTimeout ensures that other components have been rendered,
+        // which is necessary to properly position the plugin.
+        setTimeout(() => {
+          appState.document.content?.applyModelChange(() => {
+            const plugin = appState.document.content?.createTile?.(kWebViewTileType)
+            if (isWebViewModel(plugin?.content)) plugin.content.setUrl(di)
```

Comment:
FYI: You can pass a content snapshot to `createTile` which would allow you to eliminate the second line, if this code is ever revisited.